description/src/lib.rs

```diff
@@ -0,0 +1,137 @@
+mod description;
+mod formatting;
+mod transformer;
+
+#[cfg(feature = "type-example")]
+pub mod type_example;
+
+pub use description::type_description;
```

Comment:
I wonder whether it makes sense to export each of the three things here, something like:

- `scale_value(..)`/`scale_value_from_seed(..)` - Generate an example `scale_value::Value` from type info
- `rust_value(..)`/`rust_value_from_seed(..)` - Generate an example rust value from type info
- `rust_type(..)`- Generate a rust type from type info